README.md

```diff
@@ -257,7 +257,69 @@ Add your gateway by providing a name and connection profile via the `Add Gateway
 You can also create a gateway from a fabric environment. When you run the `Add Gateway` command there will be an option to create a gateway from a fabric environment, select this then choose the environment you want to create the gateway from.
 
 ### Connect to a gateway and discover its resources
-Connect by clicking on a gateway in the `Fabric Gateways` panel, and expand the navigation tree to explore its resources. Instantiated Smart Contracts are listed under the channel and from here you can generate functional-level test files on single or multiple smart contracts (Currently you cannot generate Java functional tests). Submit or evaluate individual transactions listed under the instantiated smart contracts, with the result displayed in the `Blockchain` output channel.
+Connect by clicking on a gateway in the `Fabric Gateways` panel, and expand the navigation tree to explore its resources. Instantiated Smart Contracts are listed under the channel and from here you can generate functional-level test files on single or multiple smart contracts. Submit or evaluate individual transactions listed under the instantiated smart contracts, with the result displayed in the `Blockchain` output channel.
+
+#### _<font color='red'>* Beta * </font> Java functional tests_
```

Comment:
This doesn't appear red unfortunately - https://github.com/lquintai/blockchain-vscode-extension/tree/javaFunctTests#-beta---java-functional-tests

I think it's fine if **BETA** is in bold using: `**BETA**`